apps/comments-ui/src/App.tsx

```diff
@@ -192,7 +192,7 @@ const App: React.FC<AppProps> = ({scriptTag}) => {
             <CommentsFrame ref={iframeRef}>
                 <ContentBox done={done} />
             </CommentsFrame>
-            <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/>
+            {done ? <AuthFrame adminUrl={options.adminUrl} onLoad={initAdminAuth}/> : null}
```

Comment:
My React is a little rusty - `null` won't render anything, right?

Reply:
Right, `null` in this case means render nothing.

https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null